api/v1alpha1/service_fence.proto

```diff
@@ -83,6 +83,10 @@ message ServiceFenceSpec {
     map<string, RecyclingStrategy> host = 1;
     // Switch to render servicefence as sidecar
     bool enable = 2;
+    // services in these namespaces are all stable dependency
```

Comment:
I think `static` semantic is more clear

Reply:
change annotation to static

Reply:
change to labelSelector.selectors with repeated selector (map[string]string)